gordon.toml.example

```diff
@@ -11,6 +11,7 @@ project = "my-gcp-project"
 [gcp.enricher]
 keyfile = "/path/to/keyfiles/compute.json"
 dns_zone = "example.com."
+managed_zone = "example-com"
```

Comment:
I would if instead of adding this here, should both the dns and managed zone be at the gcp level?  I don't know if this was already discussed or if I'm misunderstanding how the config gets parsed, but I can't think of a reason where you'd want the enricher to have a different value than the publisher, and having things in two places is a good way for them to get accidentally out of sync.

Reply:
Hm, okay, I had that in this PR originally but had moved it to the current configuration as a result of this comment chain: https://github.com/spotify/gordon-gcp/pull/56#discussion_r201820886

Reply:
Updated it though, as it makes sense to have them together and at a higher level (`gcp`), especially our readthedocs talk more about configuration and overriding values and etc.